### DISCUSSION TITLE
Is it possible to disable CloudFormation stack update?

### DISCUSSION THREAD

**Discussion by sorasit-teebeea** [mozilla/hubs#5095] [Hubs Cloud Questions] [answered]
Due to accident that happened on Feb 8 2022, Our customer can't access to VR scene. It show "The scene failed to load"

And after log what happened, It is 403 error in Cloudformation EFS. Which can't get .glb file

So, am asking support team is it possible to disable CloudFormation stack update?
Or you have any workaround to prevent this issue.

**Comment by pattersonbl2** [comment, marked as answer]
CloudFormation doesn't auto-update. The issue was related to users skipping an update of cloud formation four months ago. This process is manual; we try to announce this in the discord community. A few users have turned off the auto software updates, but this could also break your stack.

**Reply by sorasit-teebeea** [reply]
Thank you 

So, This manual update **must** proceed right?
Unable to skip or disable otherwise this problem will happened again.

**Reply by pattersonbl2** [reply]
We try to make things backward compatible with older stacks, but we can't guarantee this for various reasons. It's best to keep things updated to ensure everything is aligned.

**Reply by himhmt** [reply]
@Pattersonbl2 
Hello
>  A few users have turned off the auto software updates, 

Can you tell me where to find the instructions to stop the automatic updates?
